src/Shared/KustoDataTypes.cs

```diff
@@ -10,6 +10,7 @@ public static class KustoDataTypes
         public const string Long = "long";
         public const string Int = "int";
         public const string Boolean = "bool";
+        public const string Decimal = "decimal";
```

Comment:
Decimal is a very odd data type, are you sure you need this?  I think "real" is more appropriate (I'd name the string "Double" but have the string be "real", since Double is what people are expecting in C#.